### ISSUE TITLE
Export the SBAS Result in Geotiff format

### ISSUE THREAD

**Issue by ArgoBagus10** [mobigroup/YamchiDam#2]
Dear Alexey,

Thank you so much for creating PyGMTSAR for process SAR imagery.
I want to learn SBAS processing using PyGMTSAR in land subsidence and deformation.
I have a question,
Can we export the results from LOS Displacement from SBAS as a geotiff format?
So that we can process the result in QGIS GIS app and overlay that with the other map

Best regards,

Argo.

**Comment by AlexeyPechnikov** [comment]
Yes, that’s possible to export GeoTIFF files. But the best way is to open the output WGS84 NetCDF files in QGIS by the same way. QGIS app and GDAL tools have native NetCDF support for years.

**Comment by ArgoBagus10** [comment]
what is the code for exporting the results from SBAS processing in google colab and PyGMTSAR?

**Comment by AlexeyPechnikov** [comment]
See the Google Colab examples listed on GitHub: https://github.com/mobigroup/gmtsar
